### DISCUSSION TITLE
Server Migration

### DISCUSSION THREAD

**Discussion by pedrodebarros** [rustdesk/rustdesk#5974] [Q&A]
Hello!

I would like to migrate my service provider relay server to IPv6, but it would be very important to change the security keys from one server to another, so I don't need to reinstall the remote on all PCs.
I didn't find any documentation for this type of situation, does anyone have any references or tips?

Thanks

**Comment by SergeyMy** [comment]
https://github.com/rustdesk/rustdesk-server

> On container startup, the presence of the keypair is checked (/data/id_ed25519.pub and /data/id_ed25519) and if one of these keys doesn't exist, it's recreated from ENV variables or docker secrets. Then the validity of the keypair is checked: if public and private keys doesn't match, the container will stop. If you provide no keys, hbbs will generate one for you, and it'll place it in the default location.
